web/packages/teleport/src/components/TdpClientCanvas/TdpClientCanvas.tsx

```diff
@@ -97,6 +97,29 @@ function TdpClientCanvas(props: Props) {
     }
   }, [client, clientOnPngFrame]);
 
+  useEffect(() => {
+    if (client) {
+      const canvas = canvasRef.current;
+      const updatePointer = (pointer: {data: ImageData, hotspot_x: number, hotspot_y: number}) => {
```

Comment:
Technically, given how we're using it in the Rust wasm, this should be
```suggestion
      const updatePointer = (pointer: {data?: ImageData, hotspot_x?: number, hotspot_y?: number}) => {
```